### PR TITLE
Client Profile Demographic Info (DEV-167)

### PR DESCRIPTION
**Pull request by Davit-BetterAngels**
DEV-167

### CODE REVIEW

File: libs/expo/betterangels/src/lib/ui-components/GenderPicker.tsx [hunk 1]
```diff
@@ -64,6 +76,17 @@ export default function GenderPicker<T extends FieldValues>(
             control={control}
           />
         )}
+        {onReset && (
+          <View style={{ alignItems: 'flex-end' }}>
+            <TextButton
+              mt="sm"
+              color={Colors.PRIMARY}
+              title="Reset"
+              accessibilityHint="resets pronouns"
```
Comment (mikefeldberg) [inline libs/expo/betterangels/src/lib/ui-components/GenderPicker.tsx:85]:
```suggestion
              accessibilityHint="resets gender"
```

**Review by mikefeldberg — APPROVED** [review]
just one typo in an accessibility hint

**Comment by shiozakiyuko** [comment]
Hi @Davit-BetterAngels 

After I typed something in the Physical Description, and I want to get out of it, nothing happens (I can't close the keyboard) when I tapped on the title, or outside of keyboard. I moved on to Martial Status and tapped Return, the cursor is still active in Physical Description text field.

**Review by shiozakiyuko — COMMENTED** [review]
Hi @Davit-BetterAngels

I noticed a couple of things.

1. After I typed something in the Physical Description, and when I want to get out of it, nothing happens (I can't close the keyboard) when I tapped on the title, or outside of keyboard. I moved on to Martial Status and tapped Return, the cursor is still active in Physical Description text field. (I'm testing this with my iPhone)

2. I also noticed that the remove button overlaps with the text. Could you wrap the text before it reaches to the remove button section?

File: libs/expo/betterangels/src/lib/ui-components/GenderPicker.tsx (hunk 1 above)
Comment (Davit-BetterAngels) [reply libs/expo/betterangels/src/lib/ui-components/GenderPicker.tsx]: done

File: apps/betterangels/src/app/team/Edit.tsx [hunk 2]
```diff
@@ -54,7 +54,6 @@ export default function TeamEditScreen(props: ITeamEditScreenProps) {
       <Input mb="md" label="Team Name" control={control} name="name" />
       <Textarea
         mb="md"
-        height={200}
```
[the remark this answers is not in the document]
Comment (shiozakiyuko) [reply apps/betterangels/src/app/team/Edit.tsx:57]: One more thing.

The Gender - Other option should be at the bottom.

[the remark this answers is not in the document]
Comment (shiozakiyuko) [reply apps/betterangels/src/app/team/Edit.tsx:57]: Another one: I didn't see the Pronouns - Other wasn't saved after I tapped on Update button.

[the remark this answers is not in the document]
Comment (shiozakiyuko) [reply apps/betterangels/src/app/team/Edit.tsx:57]: Hi @Davit-BetterAngels 

I confirmed that the first 2 things are fixed: The text overlaps the X - remove button, and the getting out from the section issue.

Let me know the other issues when they are fixed. Thanks!

[the remark this answers is not in the document]
Comment (shiozakiyuko) [reply apps/betterangels/src/app/team/Edit.tsx:57]: Hi @Davit-BetterAngels 

Everything looks good! 

One question though. 
When I updated one information on a device, and check the info if it's updated on another device, I have to close the app once and re-open the app again in order to see the update. Is this a loading issue? Should we create ticket for this?

[the remark this answers is not in the document]
Comment (Davit-BetterAngels) [reply apps/betterangels/src/app/team/Edit.tsx:57]: we don't have live db

**Review by shiozakiyuko — APPROVED** [review]

**Pull request merged**